src/tables.js

```diff
@@ -30,6 +30,8 @@ const Flow = {
 
   /* Kerning (DEPRECATED) */
   IsKerned: true,
+  keyProperties: keyProperties,
+  integerToNote: integerToNote,
```

Comment:
This is good for now, but I'd suggest that we tackle tables.js => tables.ts (or maybe finally the time to rename to flow.ts -- I always forget where Flow is?) and make them `export function...` for each of the properties with a `export default const Flow = { }` at the end?

Reply:
I agree that Flow location is not obvious. I consider flow.ts much better but it would change the interface to apps using the code.

We will see a change here anyway if we port `tables` because TS does not like nested definitions:
`Flow.keyProperties` with a function
`Flow.keyProperties.note_values` with the values.

Reply:
This is not a `eslint` it is a `tsc` import topic. Import at compilation time will only recognize the elements declared as part of the structure, not the ones added on run-time.

I suggest to leave it.

Reply:
I agree with @mscuthbert this is good for now

Reply:
okay